src/debugger/Debugger.js

```diff
@@ -50,7 +47,7 @@ export class Debugger {
     return false;
   }
 
-  checkForBreakpoint(ast: BabelNode, contextStack: Array<ExecutionContext>) {
+  checkForBreakpoint(ast: BabelNode) {
     invariant(ast.loc);
```

Comment:
We are unable to guarantee this invariant in our code since we depend on what Bablylon comes up with. Also, the logic below ignores AST nodes that span multiple lines. This seems a bit brittle. A single assignment statement can easily span more than one source line and a block of more than one statement can easily fit into a single line.